contracts/AppCoinsIAB.sol

```diff
@@ -8,11 +8,10 @@ contract AppCoins {
 
 contract AppCoinsIABInterface {
     function division(uint numerator, uint denominator) public view returns (uint);
-    function buy(uint _amount, string _sku, address _addr_appc, address _dev, address _appstore, address _oem) public view returns (bool);
+    function buy(uint _amount, string _sku, address _addr_appc, address _dev, address _appstore, address _oem, string packageName, bytes2 countryCode) public view returns (bool);
```

Comment:
It's a minor thing but, wouldn't it be better to have package name and _sku as the first arguments of the function and the event? Or at least packageName near sku, as they are related.
Another minor detail,  packageName and countryCode should both start with underscore, for consistency.

Reply:
good point.